src/main/java/teammates/logic/core/LocalFileStorageService.java

```diff
@@ -0,0 +1,57 @@
+package teammates.logic.core;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+/**
+ * Holds functions for operations related to binary file storage in local dev environment.
+ */
+public final class LocalFileStorageService implements FileStorageService {
+
+    private static final String BASE_DIRECTORY = "../../filestorage-dev";
+
+    private static String constructFilePath(String fileKey) {
+        return BASE_DIRECTORY + "/" + fileKey;
+    }
+
+    @Override
+    public void delete(String fileKey) {
+        File file = new File(constructFilePath(fileKey));
+        file.delete();
+    }
+
+    @Override
+    public void create(String fileKey, byte[] contentBytes, String contentType) {
+        try (OutputStream os = Files.newOutputStream(Paths.get(constructFilePath(fileKey)))) {
+            os.write(contentBytes);
+        } catch (IOException e) {
+            // TODO
```

Comment:
Maybe just use the logger for this?

Reply:
I don't see it as that useful, but nothing to lose by doing it